lib/grid.rb

```diff
@@ -13,12 +13,20 @@ def template
     end
 
   def coordinate_valid?(coordinate)
-    coordinates.include?(coordinate)
+    coordinates.key?(coordinate)
   end
 
-  private
+  def mark(coordinate)
+    coordinates[coordinate] = 'X'
```

Comment:
This isn't setting the value like I expect it to. I can set a value for the key like this in the console. What am I missing here?